packages/web/src/services/audio-player/AudioPlayer.ts

```diff
@@ -70,6 +70,7 @@ class fLoader extends Hls.DefaultConfig.loader {
             this.getFallbacks(),
```

Comment:
this looks like its for segments -- is codepath shouldn't be in use anymore right?

Reply:
yeah good callout, @raymondjacobson do you think it's time for us to remove hls + track-segment references?

Reply:
yep, we should drop it all! 🪓

Reply:
sweet added https://linear.app/audius/issue/C-2429/remove-segments-hls-in-client